src/components/Dashboard/UpcomingBookings.vue

```diff
@@ -87,11 +90,11 @@ onMounted(() => {
         <button class="button red-bg" @click="router.push('/book/realtime')"> Book Slot </button>
     </div>
     <div class="observations upcoming-obs">
-        <h3><span v-if="!requestGroups.length">No </span>Pending Requests</h3>
+        <h3><span v-if="Object.keys(pendingScheduledObservations).length == 0">No </span>Pending Requests</h3>
```

Comment:
I would probably call this "Scheduled Requests" instead of "Pending Requests". You can have pending (submitted) requests that have not been scheduled and thus do not have any observations yet.